### PR TITLE
Second CL for the langsvr

### PR DESCRIPTION
**Pull request by AWoloszyn**
Please ignore the first change in the list. It exists as a separate PR already.

### CODE REVIEW

File: gapil/langsvr/analyze.go [hunk 1]
```diff
@@ -250,10 +251,12 @@ func (a *analyzer) doAnalysis(
 		defer shutdown(ctx)
 		events := &task.Events{}
 		executor := task.Batch(pool, events)
-
+		mt := sync.Mutex{}
 		for path, da := range das {
-			path, da, ctx := path, da, log.V{"file": path}.Bind(ctx)
+			path, da, _ := path, da, log.V{"file": path}.Bind(ctx)
```
Comment (ben-clayton) [inline gapil/langsvr/analyze.go]: What's the point of the 3rd assignment now?

File: tools/build/third_party/vscode-languageserver-types.BUILD [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2018 Google Inc.
```
Comment (ben-clayton) [inline tools/build/third_party/vscode-languageserver-types.BUILD]: Wrong year

File: tools/build/workspace.bzl [hunk 3]
```diff
@@ -200,6 +200,33 @@ def gapid_dependencies(android = True, mingw = True, locals = {}):
...
+        sha256 = "42ad6dc73bbf24a067d1e21038d35deab975cb207ac2d63b81c37a977d431d8f",
+    )
+
+    maybe_repository(
+        http_archive,
+        name = "vscode-jsonrpc",
+        locals = locals,
+        url = "https://registry.npmjs.org/vscode-jsonrpc/-/vscode-jsonrpc-2.4.0.tgz",
+        build_file = "@gapid//tools/build/third_party:vscode-jsonrpc.BUILD",
+         sha256= "bed9b2facb7d179f14c8a710db8e613be56bd88b2a75443143778813048b5c89",
```
Comment (ben-clayton) [inline tools/build/workspace.bzl]: Nit: Stray whitespace

File: gapil/langsvr/analyze.go (hunk 1 above)
Comment (AWoloszyn) [reply gapil/langsvr/analyze.go]: Good point!

**Pull request merged**